lake-manifest.json

```diff
@@ -4,7 +4,7 @@
  [{"url": "https://github.com/leanprover/std4",
    "type": "git",
    "subDir": null,
-   "rev": "d8610e1bcb91c013c3d868821c0ef28bf693be07",
```

Comment:
For reference: https://github.com/leanprover/std4/compare/d8610e1bcb91c013c3d868821c0ef28bf693be07...0f6bc5b32bf5b0498902d3b5f0806c75530539d5